torch/_inductor/codecache.py

```diff
@@ -1599,9 +1599,11 @@ def compile(cls, source_code, dst_file_ext) -> Tuple[str, str, str]:
             with lock:
                 output_path = input_path[: -len(cls._SOURCE_CODE_SUFFIX)] + dst_file_ext
                 if not os.path.exists(output_path):
-                    cmd = cuda_compile_command(
+                    cmdstr = cuda_compile_command(
```

Comment:
This is just here for development purposes / debugging. Not part of any final PR.